card/mod/bootstrap/set/abstract/bootswatch_theme.rb

```diff
@@ -45,13 +45,13 @@
 
 PRE_VARIABLES_CARD_NAMES = %i[
   style_jquery_ui_smoothness
-  style_cards
   style_right_sidebar
   bootstrap_functions
 ].freeze
 
 POST_VARIABLES_CARD_NAMES = %i[
   bootstrap_core
+  style_cards
```

Comment:
how bad is it that this line isn't in 0.5?

Reply:
no problem. we need that only for #248